### PR TITLE
Update designer/developer navigation & pages

### PR DESCRIPTION
**Pull request by corcorankt1**
---
name: Pull Request
about: Create a pull request to make changes to Spark.

---

## What does this PR do?
Removes the outdated Getting Started - Designers section and breaks the Developer section into two separate Vanilla and Angular pages.

## Please check off completed items as you work.
If a checklist item or section does not apply to your PR
then please remove it.
### Documentation
 - [x] Update Spark Docs Angular
 - [x] Update Spark Docs Vanilla
 - [ ] Update Component Sass Var/Class Modifier table

### Code
 - [ ] Build Component in Spark Vanilla (Sass, HTML, JS)
 - [ ] Build Component in Spark Angular
 - [ ] Unit Testing in Spark Vanilla (100% coverage, 100% passing)
 - [ ] Unit Testing in Spark Angular (100% coverage, 100% passing)

### Accessibility
- [ ] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)

### Browser Testing (current version and 1 prior)
  - [ ] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)

### Design Review
 - [ ] Design reviewed and approved

### Screenshots
Add screenshots to help explain your PR if you'd like. However, this is not
expected.

### CODE REVIEW

File: src/pages/gettingstarted/angular.hbs [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+            <span class="drizzle-c-Command">
+              <code class="drizzle-CodeHighlight">
+                BrowserAnimationsModule
+              </code>
+            </span>
+          to the NgModule imports array.
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>Spark does not provide icons directly. What you see below are proprietary icons in use by Quicken Loans. To supply
```
Comment (RCopeland) [inline src/pages/gettingstarted/angular.hbs]: can you change it from see below to like 'see on our icons page'?

File: src/pages/gettingstarted/vanilla.hbs [hunk 2]
```diff
@@ -0,0 +1,313 @@
+---
+title: Vanilla
```
Comment (RCopeland) [inline src/pages/gettingstarted/vanilla.hbs]: maybe we should change Vanilla to HTML, i dont think its a great name, but it matches the code switch

File: src/pages/gettingstarted/vanilla.hbs [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+    </ul>
+  </div>
+
+  <div class="drizzle-o-ContentGrouping">
+    <h2 class="drizzle-b-h2">Spark Extras</h2>
+
+    <ul class="drizzle-b-List drizzle-b-List--spacing-m drizzle-b-List--nested">
+      <li>
+        <p>There are a few Spark Extras patterns available for install.</p>
+        <div class="drizzle-c-Command">
```
Comment (RCopeland) [inline src/pages/gettingstarted/vanilla.hbs]: can you put these in a list?

File: src/pages/gettingstarted/angular.hbs [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+            <code class="drizzle-CodeHighlight">
+              spark-core-angular
+            </code>
+          </span>
+          .
+        </p>
+      </li>
+      <li class="sprk-u-mvm">
+        <p>
+          Install additional peer dependencies:
```
Comment (RCopeland) [inline src/pages/gettingstarted/angular.hbs]: these are listed as dev deps in the spark-core-angular project, so they shouldnt have to manually install them

File: src/pages/gettingstarted/angular.hbs [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+    <div class="drizzle-o-ContentGrouping">
+      <h2 class="drizzle-b-h2">Change Workflow and Contribution Guide</h2>
+      <p>If you are interested in contributing to Spark, please read our full
+        <a class="drizzle-b-Link" href="/docs/change-workflow">change
+          workflow</a> and <a class="drizzle-b-Link" href="/docs/contributions">
+          contribution guide</a> to get started.</p>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h3 class="drizzle-b-h3">Code Style Standards</h3>
```
Comment (RCopeland) [inline src/pages/gettingstarted/angular.hbs]: i think we should move the code style stuff to the contributions page

File: src/pages/gettingstarted/angular.hbs (hunk 1 above)
Comment (corcorankt1) [reply src/pages/gettingstarted/angular.hbs]: Updated

File: src/pages/gettingstarted/vanilla.hbs (hunk 2 above)
Comment (corcorankt1) [reply src/pages/gettingstarted/vanilla.hbs]: Updated to html throughout

File: src/pages/gettingstarted/vanilla.hbs (hunk 3 above)
Comment (corcorankt1) [reply src/pages/gettingstarted/vanilla.hbs]: Added a list

File: src/pages/gettingstarted/angular.hbs (hunk 4 above)
Comment (corcorankt1) [reply src/pages/gettingstarted/angular.hbs]: Removed reference to additional dependencies

File: src/pages/gettingstarted/angular.hbs (hunk 5 above)
Comment (corcorankt1) [reply src/pages/gettingstarted/angular.hbs]: Removed from html and angular doc pages, moved to contributions.

**Pull request merged**